test/extended/util/oauthserver/oauthserver.go

```diff
@@ -491,8 +493,8 @@ func determineImageFromRelease(oc *exutil.CLI) (string, error) {
 	return util.DetermineImageFromRelease(oc, "oauth-server")
 }
 
-func newRequestTokenOptions(config *restclient.Config, oauthServerURL, oauthClientName, username, password string) *tokencmd.RequestTokenOptions {
-	options := tokencmd.NewRequestTokenOptions(config, nil, username, password, false)
+func newRequestTokenOptions(config *restclient.Config, oauthServerURL, oauthClientName, username, password string) *tokenrequest.RequestTokenOptions {
+	options := tokenrequest.NewRequestTokenOptions(config, false, challengehandlers.NewBasicChallengeHandler("", os.Stdin, os.Stdout, nil, username, password))
```

Comment:
Why do we now use `""` as the host arg when creating a `BasicChallengeHandler`? The previous implementation would use `config.Host` instead.

Reply:
I see. It shouldn't be relevant to how it's used but I'll fix it.